Tools/clinic/clinic.py

```diff
@@ -4227,6 +4226,9 @@ def state_modulename_name(self, line):
         module, cls = self.clinic._module_and_class(fields)
 
         fields = full_name.split('.')
+        if fields[-1] in unsupported_special_methods:
```

Comment:
Dear reviewers,
This change is intended to early detect the consistency of `unsupported_special_methods`.